src/pzt/mysql/sections/10-try/10-certificate.mdx

```diff
@@ -0,0 +1,55 @@
+<p>
+  Before you can configure {props.pzt.name} TLS, you will need a certificate
+  issued by a trusted certificate authority (CA). If you already have a
+  certificate, private key, and CA root certificate from your organization's
+  existing CA, you can skip to the {props.pzt.name} TLS configuration section
+  below. If you need to generate a certificate, you can:
+</p>
+
+<ul>
+  <li>
+    Use a{' '}
+    <a href="/signup?product=cm">
+      free hosted Smallstep Certificate Manager authority
+    </a>
+  </li>
+  <li>
+    Roll your own with{' '}
+    <a href="/docs/step-ca">
+      open source <inlineCode>step-ca</inlineCode>
+    </a>
+  </li>
+</ul>
+
+<Alert severity="warning">
+  This tutorial is designed for private networks, not publicly trusted websites
+  or endpoints. If you are looking to automate certificates for public websites,
+  servers, or endpoints, we recommend using{' '}
+  <a href="https://letsencrypt.org/">Let's Encrypt</a> and the ACME instructions
+  below.
+</Alert>
+
+<p>
+  To request a certificate from your CA using the{' '}
+  <a href="https://smallstep.com/docs/step-cli">
+    <inlineCode>step</inlineCode>
+  </a>{' '}
+  CLI, bootstrap your CA with <inlineCode>step ca bootstrap</inlineCode> and run
+  the following command (sub the server name for the actual name / DNS name of
+  your {props.pzt.name} server).
+
+</p>
+
+<MDXBlock path="snippets/certificate" />
+
+Note that MySQL uses RSA keys, thus the `--kty=RSA` here.
+Your certificate and private key will be saved in `server-cert.pem` and `server-key.pem`, respectively.
+
+MySQL is one of a few services that require the intermediate CA certificate to be bundled with the root CA certificate, in order to validate certificate chains. Let's create a bundle of your CA intermediate and root certificates:
+
+```shell
+cat server-cert.pem | awk '/BEGIN/,/END    /{ if(/BEGIN/){a++}; if (a==2) {print} }' > ca.pem
+step ca root >> ca.pem
+```
+
+The certificate bundle will be saved in `ca.pem`.
```

Comment:
This looks like a big copy/paste from the default. Is it just the bottom section that changed? If so, I think maybe it makes more sense to use:

```jsx
<MDXBlock default path="sections/10-try/10-certificate" />

... additional MySQL content here ...
```

Reply:
Or possibly add the caveat about the `--kty=RSA` flag into `snippets/certificate` instead of `10-certificate`? I pretty much has MySQL and RSA in mind when making that certificates snippet.

Reply:
unfortunately the snippet is not enough, because I need to use unconventional filenames for the cert and key, and the default has `server.crt` and `server.key` hardcoded. Honestly I don't mind a bit of duplication here, but I think I'm not as much of a stickler about DRYing things up as you are. No right or wrong here though, it's really a matter of preference.

Reply:
oh and the `ca.crt` filename is hardcoded too, but I need it to be `ca.pem` (mysql convention)